wolfssl/ssl.h

```diff
@@ -45,6 +45,10 @@
     #endif
 #endif
 
+#if !defined(NO_ASN) && defined(WOLFSSL_CERT_SIGNER_INFO)
+    #include <wolfssl/wolfcrypt/asn_public.h>
```

Comment:
I'd prefer to keep wolfssl/ssl.h opaque.  That is, no use of types.h or internal headers.  Why is asn_public required?